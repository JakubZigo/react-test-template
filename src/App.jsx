import React, { Component } from 'react'
import Foo from './Foo.jsx'
import YesNo from './YesNo.jsx'

class App extends Component {
  render() {
    return (
      <div>
        <Foo />
        <YesNo />
      </div>
    )
  }
}

export default App

