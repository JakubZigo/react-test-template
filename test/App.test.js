import React from 'react'
import { shallow, mount, render } from 'enzyme'

import App from '../src/App'

describe('App', () => {
  const wrapper = shallow(<App />)

  it('Should render Foo', () => {
    expect(wrapper.find('Foo').length).toBe(1)
  })
})
