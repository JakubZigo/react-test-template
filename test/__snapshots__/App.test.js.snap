// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<div>
  <Connect(Foo) />
  <Connect(YesNo) />
</div>
`;
